Packs/SecurityScorecard/Integrations/SecurityScorecard/SecurityScorecard.yml

```diff
@@ -0,0 +1,421 @@
+      default: false
+    - name: timing
+      description: Timing granularity. Acceptable values are `weekly` or `daily`
+      required: false
+      auto: PREDEFINED
+      predefined:
+      - "daily"
+      - "weekly"
+    outputs:
+    - contextPath: SecurityScorecard.Company.History.domain
+      description: Company domain.
+      type: String
+    - contextPath: SecurityScorecard.Company.History.date
+      description: Score date.
+      type: Date
+    - contextPath: SecurityScorecard.Company.History.score
+      description: Company historical security score in numeric form (55-100)
+      type: Number
+  - name: securityscorecard-company-history-factor-score-get
+    description: Retrieve company historical factor scores
+    arguments:
+    - name: domain
+      description: Company domain, e.g. google.com
+      required: true
+      default: false
+    - name: from
+      description: Initial date for historical data. Value should be in format 'YYYY-MM-DD'
+      required: false
+      default: false
+    - name: to
+      description: Initial date for historical data. Value should be in format 'YYYY-MM-DD'
+      default: false
+    - name: timing
+      description: Timing granularity. Acceptable values are "daily" (default), "weekly"
+        or "monthly"
+      required: false
+      auto: PREDEFINED
+      predefined:
+      - "daily"
+      - "weekly"
+      - "monthly"
+    outputs:
+    - contextPath: SecurityScorecard.Company.FactorHistory.domain
+      description: Company domain.
+      type: String
+    - contextPath: SecurityScorecard.Company.FactorHistory.date
+      description: Score date.
+      type: Date
+    - contextPath: SecurityScorecard.Company.FactorHistory.Factor.name
+      description: Factor name.
+      type: Number
+    - contextPath: SecurityScorecard.Company.FactorHistory.score
+      description: Company historical security score in numeric form (55-100)
+      type: Number
+  - name: securityscorecard-alert-grade-change-create
+    description: Create alert based on grade
+    arguments:
+    - name: change_direction
+      description: Direction of change. Possible values are 'rises' or 'drops'.
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - "rises"
+      - "drops"
+    - name: score_types
+      isArray: true
+      description: Types of risk factors to monitor. Possible values are 'overall',
+        'any_factor_score', 'network_security', 'dns_health', 'patching_cadence',
+        'endpoint_security', 'ip_reputation', 'application_security', 'cubit_score',
+        'hacker_chatter', 'leaked_information', 'social_engineering'. For multiple
+        factors, leaked_information,social_engineering .
+      required: true
+    - name: target
+      description: What do you want to monitor with this alert. It could be one of
+        the following 'my_scorecard', 'any_followed_company'. This argument is require
+        if the `portfolios` argument is not specified.
```

Comment:
```suggestion
      description: What do you want to monitor with this alert. It could be either 'my_scorecard' or 'any_followed_company'. This argument is required
        if the `portfolios` argument is not specified.
```